README.md

```diff
@@ -34,17 +40,47 @@ dev_dependencies:
   flappy_translator: 
 ```
 
+### Define Settings
+
+Settings for *flappy_translator* can be optionally set in your project's *pubspec.yaml* file:
+
+```yaml
+flappy_translator:
+  input_file_path: "test.csv"
+  output_dir: "lib"
+  file_name: "i18n"
+  class_name: "I18n"
+  delimiter: ","
+  start_index: 1
+  depend_on_context: true
+  use_single_quotes: false
```

Comment:
I tried to put `use_single_quotes` to `true` but it didn't work..
Here is my CSV test :

```
keys;Description;fr;en
title;le titre;"Salut, ""c'est moi""";"Hey, ""it's me"""
```

(with delimite `;` here, but it does not seem to change anything

Reply:
Here the `'` character in the input string needs to be escaped, 

```
keys;Description;fr;en
title;le titre;"Salut, ""c\'est moi""";"Hey, ""it\'s me"""
```

If using double quotes, then this character would not need to be escaped.

In this case I assumed that the user would already escape all single quotes if they wanted to export the strings using single quotes, for instance

```
keys;Description;fr;en
title;le titre;Salut, "c\'est moi";Hey, "it\'s me"
```

I could add a check that if `use_single_quotes` is true, then all `'` characters in the string are automatically escaped. This seems like a useful feature.

Reply:
After some more testing, your initial example

```
keys;Description;fr;en
title;le titre;"Salut, ""c'est moi""";"Hey, ""it's me"""
```

exported correctly with the settings

```yaml
flappy_translator:
  input_file_path: "test.csv"
  output_dir: "lib"
  file_name: "i18n"
  class_name: "I18n"
  delimiter: ";"
  start_index: 2
  depend_on_context: true
  use_single_quotes: true
```

(note that start index is 2 because of description column).

I've added a new check that will escape `'` when `use_single_quotes` is true. Thus even

```
keys;Description;fr;en
title;le titre;Salut, c'est moi;Hey, it's me
```

will export correctly. Please let me know if you find any issues.